modules/claims_api/spec/requests/v2/veterans/rswag_526_spec.rb

```diff
@@ -84,8 +84,7 @@ def append_example_metadata(example, response)
         },
         'Maximum Attributes' => {
           value: JSON.parse(Rails.root.join('modules', 'claims_api', 'spec', 'fixtures', 'v2', 'veterans',
-                                            'disability_compensation', 'form_526_json_api.json').read)
-
+                                            'disability_compensation', 'valid_526_maximum.json').read)
```

Comment:
@stiehlrod  I suspect the new fixture is probably the way to go. Do we also want this removed from the 202 example below this in the docs?  

![Screenshot 2024-10-08 at 2 02 50 PM](https://github.com/user-attachments/assets/e2f4f7e8-44db-4e6d-9a64-40ab1e04ed48)

It is also still in the synchronous endpoint example (and 202) so also wondering if that needs to be removed

![Screenshot 2024-10-08 at 2 04 19 PM](https://github.com/user-attachments/assets/ae49d210-8942-420d-87dc-1f770685467a)

![Screenshot 2024-10-08 at 2 02 50 PM](https://github.com/user-attachments/assets/20ea9476-ce9f-480f-bdbd-24833fb81379)